backend/backend/settings.py

```diff
@@ -85,8 +98,7 @@
         "ENGINE": "djongo",
         "NAME": "Cluster0",
         "CLIENT": {
-            "host": "mongodb+srv://lioncat:honk@cluster0.2lgaq4p.\
-                    mongodb.net/?retryWrites=true&w=majority",
```

Comment:
Invalidate these credentials,since these will be in public.

Reply:
working on it

Reply:
changed